Wolox Tranining App/ViewControllers/NewsCollectionViewController.m

```diff
@@ -0,0 +1,79 @@
+//
+//  NewsViewController.m
+//  Wolox Tranining App
+//
+//  Created by Francisco Depascuali on 5/5/15.
+//  Copyright (c) 2015 Francisco Depascuali. All rights reserved.
+//
+
+#import "NewsCollectionViewController.h"
+#import "NewsViewModel.h"
+#import "SingleNewTableViewCell.h"
+#import "NewsCollectionViewModel.h"
+#import "NewsViewModel.h"
+#import <SDWebImage/UIImageView+WebCache.h>
+
+#define CELL_ID @"NewsTableViewCell"
+
+@interface NewsCollectionViewController ()
+
+@property (nonatomic,strong) NewsCollectionViewModel* model;
```

Comment:
This variable name is not self descriptive, i'd call it newsCollectionViewModel.